src/prometheus-net.Contrib/Diagnostics/AspNetCoreListenerHandler.cs

```diff
@@ -22,12 +22,37 @@ private static class PrometheusCounters
             public static readonly Counter AspNetCoreRequestsErrors = Metrics.CreateCounter(
                 "aspnetcore_requests_errors_total",
                 "Total HTTP requests received errors.");
+
+            public static readonly Counter AspNetCoreRequestsTotal = Metrics.CreateCounter(
```

Comment:
I think we don't need this metric.

Reply:
It is also present here, though https://github.com/alexvaluyskiy/prometheus-net-contrib/blob/bd055c5114538a3cde76465640178c6d0b11751b/src/prometheus-net.Contrib/EventListeners/Adapters/PrometheusAspNetCoreCounterAdapter.cs#L10. I find it clearer to have a separate dedicated metric for this, personally.